WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -168,17 +168,21 @@ class PostCoordinator: NSObject {
 
     private func updateReferences(to media: Media, in post: AbstractPost) {
         guard var postContent = post.content,
+            let mediaID = media.mediaID?.intValue,
             let remoteURLStr = media.remoteURL else {
             return
         }
 
         let mediaUploadID = media.uploadID
+        let gutenbergMediaUploadID = media.gutenbergUploadID
         if media.remoteStatus == .failed {
             return
         }
         if media.mediaType == .image {
             let imgPostUploadProcessor = ImgUploadProcessor(mediaUploadID: mediaUploadID, remoteURLString: remoteURLStr, width: media.width?.intValue, height: media.height?.intValue)
             postContent = imgPostUploadProcessor.process(postContent)
+            let gutenbergImgPostUploadProcessor = GutenbergImgUploadProcessor(mediaUploadID: gutenbergMediaUploadID, serverMediaID: mediaID, remoteURLString: remoteURLStr)
```

Comment:
Is this replacing the post content for _all_ posts, not just those in Gutenberg?

Reply:
So we apply both processors on all posts, but only the GB ones will be affected for the processor, because they have a different format for the media uploads ids.